clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -166,9 +166,9 @@
 
     static {
         CONFIG = new ConfigDef().define(BOOTSTRAP_SERVERS_CONFIG,
-                                        Type.LIST,
-                                        Importance.HIGH,
-                                        CommonClientConfigs.BOOSTRAP_SERVERS_DOC)
+                Type.LIST,
```

Comment:
Not intentional, just Intellij getting a little aggressive trying to "fix" the indentation. I'll fix it in the next commit.